jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1576,6 +1578,79 @@ void java_bytecode_parsert::relement_value_pair(
   }
 }
 
+/// Corresponds to the element_value structure
+/// Described in Java 8 specification 4.7.6
+/// https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.7.6
+/// Parses the bytes of the InnerClasses attribute for the current parsed class,
```

Comment:
Nit: fewer words "Parses ~the bytes of~ the InnerClass attribute"

Reply:
Clarity: current parsed class or current parsed class file? (If the former, ignore me)